live-examples/js-examples/typedarray-subarray.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js" class="language-js">
+var uint8 = new Uint8Array([ 10, 20, 30, 40, 50]);
```

Comment:
For consistency with the other examples, start this line on the same line as `<code>` above

Reply:
Sure! For some reason it didn't show me the giveaway blank line above it in the editor.